rcgen/src/lib.rs

```diff
@@ -309,20 +307,22 @@ impl DistinguishedName {
 		Self::default()
 	}
 	/// Obtains the attribute value for the given attribute type
-	pub fn get(&self, ty: &DnType) -> Option<&DnValue> {
-		self.entries.get(ty)
+	pub fn get(&self, ty: &DnType) -> Vec<&DnValue> {
```

Comment:
Isn't this a breaking API change? That conclusion also seems to be supported by the necessity to change the example code on `push()`.

Reply:
Yup, it is -- sorry I missed that.